### ISSUE TITLE
Decimal Values on Axis Tick

### ISSUE THREAD

**Issue by nirmal93guna** [naver/billboard.js#1022]
## Description
Decimal Values are appearing on the Axis values when count is set.

## Steps to check or reproduce
In the chart that I'm working on, I have to equal the y-axis and y2-axis tick count, so that the y-axis grid lines appear at the same intervals for both the axis.
To do that I'm providing the _axis.y.tick.count_ and _axis.y2.tick.count_ as 10.
To calculate exactly ten values, the axis values are left with deciomal points which I need to avoid. Is there a way to solve this problem.

One solution is to use the format function and round off the values. But this creates another problem, where 1 and 1.4 gets rounded off to 1 and 1.
Please refer the following code pen to understand the problem better.

https://codepen.io/nirmal93guna/pen/joqLPg

**Comment by netil** [comment]
Hi @nirmal93guna, the ticks are always based on the data bound.

If you don't want long decimal values are shown, you can format to be fixed.
```js
  axis: {
    y: {
      tick: {
        count: 10,
        format: y => y % 1 > 0 ? y.toFixed(2) : y
      }
    },
```

![image](https://user-images.githubusercontent.com/2178435/62849810-08730380-bd1c-11e9-8a96-43b3622f9348.png)

Or if you know the y axis boundary, you can try with the [axis.y.tick.values](https://naver.github.io/billboard.js/release/latest/doc/Options.html#.axis%25E2%2580%25A4y%25E2%2580%25A4tick%25E2%2580%25A4values) option.
But this not guarantees to be aligned with the `y2` ticks.

Basically isn't good idea on manipulate some numbers by different one, because this will alter the "real" data giving unexpected knowledge who consume it.

**Comment by nirmal93guna** [comment]
I would have to go with reduced decimal places. Calculating the axis values myself, I would have to iterate through the data to find the maxValue. I would better avoid that.

Thank you for your reply. Really appreciate your efforts. :)
